### ISSUE TITLE
Cannot add any product to cart.

### ISSUE THREAD

**Issue by PCGeekBrain** [VirtoCommerce/vc-storefront-core#237]
Version info: 
- Browser version: Google Chrome: 73.0.3683.103 
- Platform version: 2.13.43
- Storefront version: v3.2.9

### Expected behavior

Item gets added to cart or at least endpoint (`storefrontapi/cart/items`) controller (`ApiCartController.AddItemToCart`) gets hit

### Actual behavior

400 Access denied error and controller is never called (break-point on first line never hit)

### Steps to reproduce
1. Setup latest platform in IIS
2. Setup sample data
3. Clone storefront
4. Rebuild solution
5. Debug VirtoCommerce.Storefront package
6. Attempt to add any item to cart.

Possible cause:

payload sent to `storefrontapi/cart/items` looks like this in the chrome network tab:
```
{
    id: "f9330eb5ed78427abb4dc4089bc37d9f",
    quantity: 1
}
```

However the controller function (`ApiCartController.AddItemToCart`) expects a `AddCartItem` which has many more fields then this.

**Comment by yecli** [comment]
@PCGeekBrain
Checked with latest storefront master branch version - 3.2.9, breakpoint in AddItemToCart worked.
![image](https://user-images.githubusercontent.com/15610352/56024923-84d3e680-5d2a-11e9-9a6d-e4b504d70186.png)
That payload is valid for the request.
Seems something in your environment makes that happen.

**Comment by PCGeekBrain** [comment]
Weird. It does not get hit by me. 

![not-hit](https://user-images.githubusercontent.com/12478828/56039280-f186bb00-5d01-11e9-850f-b07c64243cf1.PNG)

Doing further research it seems that the `[ValidateAntiForgeryToken]` decorator is what is causing the issue. Disabling that "fixes" it. Any ideas on how that could work in your environment and not mine?

**Comment by yecli** [comment]
Don't you have Cookies disabled, or some filtering software that could modify the requests?
Also you could try to clear cookies.

**Comment by yecli** [comment]
I think I had the same issue when I got multiple storefront instances on one host (localhost with different ports or aliases) - there could be cookies mess-up.

**Comment by yecli** [comment]
@PCGeekBrain Is problem still here?

**Comment by PCGeekBrain** [comment]
@yecli sorry for the delay, been too busy with other work to get back to this integration.

I tested it out and it seems you are right. Apparently an invalid token is being saved in google chrome. Likely by the platform running in IIS on the same machine. To get around this I set it to debug in a different browser (firefox) and used chrome for the platform dashboard.

Thank you for all your help and being so understanding.
